src/ts/interfaces.ts

```diff
@@ -137,6 +137,12 @@ enum Model {
   User2Team = 'user2team',
 }
 
+enum SubModel {
```

Comment:
the name of the enum is very generic but contains only links. Either add all submodels or rename it to LinksSubmodel (preferred)